app-android/src/main/java/com/ustadmobile/port/android/view/CourseDiscussionEdit2Fragment.kt

```diff
@@ -0,0 +1,356 @@
+package com.ustadmobile.port.android.view
```

Comment:
The posts themselves should not be in the edit. The edit is only allowing the course leader to change the title, description, and don't show before date.

There is a CourseBlockEdit function that can be used (it is used by ContentEntryEdit). 

The entity being edited should actually be the CourseBlock. You can't have a Discussion without the courseblock, so we might as well use those fields for title / description.